### PR TITLE
InputMask: add new MaskPlaceholder parameter

### LINKED ISSUE #5560: [Bug]: InputMask and placeholder
### Blazorise Version

latest

### What Blazorise provider are you running on?

None

### Link to minimal reproduction or a simple code snippet

N/a

### Steps to reproduce

`<InputMask ShowMaskOnFocus="false" ShowMaskOnHover="false" Mask="****-****-****" Background="Background.White" Placeholder="Enter HUID (e.g. hdc1-tst1-hcu1)" />`

### What is expected?

Shouldn't use placeholder as mask

### What is actually happening?

The placeholder become a mask it's totally wtf

### What browsers do you see the problem on?

_No response_

### Any additional comments?

_No response_

### PR DESCRIPTION
**Pull request by stsrki**
Closes #5560

### CODE REVIEW

File: Documentation/Blazorise.Docs/Pages/Docs/Components/InputMasks/InputMaskPage.razor [hunk 1]
```diff
@@ -26,8 +26,12 @@
     </UnorderedListItem>
 </UnorderedList>
 
+<DocsPageSubtitle>
+    Examples
+</DocsPageSubtitle>
+
 <DocsPageSection>
-    <DocsPageSectionHeader Title="Basic Example">
```
Comment (David-Moreira) [inline Documentation/Blazorise.Docs/Pages/Docs/Components/InputMasks/InputMaskPage.razor:30]: We do use the Example suffix in more places, if we now don't like this, should it be changed widely?

File: Source/Blazorise/Components/InputMask/InputMask.razor.cs [hunk 2]
```diff
@@ -71,7 +71,7 @@ protected override async Task OnFirstAfterRenderAsync()
             Alias,
             InputFormat,
             OutputFormat,
-            Placeholder,
```
Comment (David-Moreira) [inline Source/Blazorise/Components/InputMask/InputMask.razor.cs:74]: Strange, I see this Placeholder being removed here, but I don't see a property reference being removed? Am I missing something?

**Review by David-Moreira — APPROVED** [review]

File: Source/Blazorise/Components/InputMask/InputMask.razor.cs (hunk 2 above)
Comment (David-Moreira) [reply Source/Blazorise/Components/InputMask/InputMask.razor.cs:74]: Or is it staying with a different purpose?

Comment (stsrki) [reply Source/Blazorise/Components/InputMask/InputMask.razor.cs:74]: MaskPlaceholder is now passed to JS lib.

File: Documentation/Blazorise.Docs/Pages/Docs/Components/InputMasks/InputMaskPage.razor (hunk 1 above)
Comment (David-Moreira) [reply Documentation/Blazorise.Docs/Pages/Docs/Components/InputMasks/InputMaskPage.razor:30]: My bad, I was under the impression we used Example alot more.

File: Source/Blazorise/Components/InputMask/InputMask.razor.cs (hunk 2 above)
Comment (David-Moreira) [reply Source/Blazorise/Components/InputMask/InputMask.razor.cs:74]: I see that. I meant that `Placeholder` was removed, but I guess it now serves a diffrent purpose, which is not the same as the maskplaceholder.

Comment (stsrki) [reply Source/Blazorise/Components/InputMask/InputMask.razor.cs:74]: Yes. Now, it is only applied directly to the input.

**Pull request merged**